docs/content/Getting-Started/quickstart.md

```diff
@@ -366,7 +375,14 @@ For `florin`:
 ```shell
 curl http://localhost:8094
 ```
-which should yield:
+
+which may yield the error below depending how long it takes for Apache HTTP Server pod to get synchronized and running:
```

Comment:
suggestion:

which may yield the error below, depending on how long it takes for the Apache HTTP Server pod to get synchronized and running: